src/Admin/entrypoint.sh

```diff
@@ -5,52 +5,27 @@
 GROUPNAME="bitwarden"
 USERNAME="bitwarden"
 
-CURRENTGID=`getent group $GROUPNAME | cut -d: -f3`
-LGID=${LOCAL_GID:-999}
+LUID=${LOCAL_UID:-0}
+LGID=${LOCAL_GID:-0}
 
-NOUSER=`id -u $USERNAME > /dev/null 2>&1; echo $?`
-LUID=${LOCAL_UID:-999}
+# Step down from host root to well-known nobody/nogroup user
 
-# Step down from host root
-
-if [ $LGID == 0 ]
+if [ $LUID -eq 0 ]
 then
-    LGID=999
+    LUID=65534
 fi
-
-if [ $LUID == 0 ]
+if [ $LGID -eq 0 ]
```

Comment:
Can we switch these back to `== 0` to be consistent with equality checks in other scripts? I prefer that syntax when possible

Reply:
The correct syntax with `[` `]` for integer comparison is `-eq` (see `man test`).
Using `bash`, we should rather use `[[` `]]` which are built-in, compared to `[` `]` which are external commands.
with `[[` `]]` we use `==`.
I however sticked to `[` `]`, as you first used them :)